pkg/schedule/coordinator.go

```diff
@@ -313,6 +312,42 @@ func (c *Coordinator) drivePushOperator() {
 	}
 }
 
+// driveSlowNodeScheduler is used to enable slow node scheduler when using `raft-kv2`.
+func (c *Coordinator) driveSlowNodeScheduler() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	ticker := time.NewTicker(time.Minute)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-c.ctx.Done():
+			log.Info("drive slow node scheduler is stopped")
+			return
+		case <-ticker.C:
+			{
+				// If enabled, exit.
+				if exists, _ := c.schedulers.IsSchedulerExisted(schedulers.EvictSlowTrendName); exists {
+					return
+				}
+				// If the cluster was set up with `raft-kv2` engine, this cluster should
+				// enable `evict-slow-trend` scheduler as default.
+				if c.GetCluster().GetStoreConfig().IsRaftKV2() {
```

Comment:
this is unbelievable

Reply:
Nope, it will not influence the current strategy when initializing.